src/test/java/se/sundsvall/supportmanagement/api/NotificationsReadResourceFailureTest.java

```diff
@@ -0,0 +1,14 @@
+package se.sundsvall.supportmanagement.api;
+
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import se.sundsvall.supportmanagement.Application;
+
+@SpringBootTest(classes = Application.class, webEnvironment = RANDOM_PORT)
+@ActiveProfiles("junit")
+public class NotificationsReadResourceFailureTest {
+
```

Comment:
Should this be implemented in another story? 
If yes: Maybe add a `//TODO:` ?

Reply:
Latest changes were not commited. File should contain tests now.